src/config.js

```diff
@@ -1,3 +1,12 @@
+
+function normalizeAccessToken(config) {
+  // eslint-disable-next-line
+  console.warn('[ShopifyBuy] accessToken is deprecated as of v1.0, please use storefrontAccessToken instead.');
+  const accessToken = config.storefrontAccessToken || config.accessToken;
```

Comment:
Is there a point in this anymore?

This function is only called when the config has an `accessToken` key.

Might as well just handle this directly below:

```js
 constructor(attrs) {
    this.requiredProperties.forEach((key) => {
      if (attrs.hasOwnProperty(key)) {
        this[key] = attrs[key];
      } else if (key === 'accessToken') {
        console.warn('[ShopifyBuy] accessToken is deprecated as of v1.0, please use storefrontAccessToken instead.');
        this.storefrontAccessToken = attrs['accessToken'];
      } else {
        throw new Error(`new Config() requires the option '${key}'`);
      }
    });
  }
```